docs/sources/developers/plugins/migration-guide/angular-react/_index.md

```diff
@@ -13,31 +13,17 @@ weight: 1000
 
 # Migrate a plugin from AngularJS to React
 
-If you’re looking to migrate a plugin to the new plugin platform, then we recommend that you release it under a new major version. Consider keeping a release branch for the previous version to be able to roll out patch releases for versions prior to Grafana 7.
+If you want to migrate a plugin to Grafana's React-based plugin platform, then we recommend that you release it under a new major version. Consider keeping a release branch for the previous version to be able to roll out patch releases for versions prior to Grafana 7.
 
-While there's no 1-to-1 migration path from an Angular plugin to the new React platform, from early adopters, we’ve learned that one of the easiest ways to migrate is to:
+While there's no standard migration path from an Angular plugin to the new React platform, we’ve learned that one of the easiest ways to migrate is to:
 
 1. Create a new branch called `migrate-to-react`.
-1. Start from scratch with one of the templates provided by Grafana Toolkit.
+1. Start from scratch with one of the templates provided by the Grafana Toolkit. SHOULDN'T THIS BE THE CREATE PLUGIN TOOL?
```

Comment:
I think there is something that went wrong on this line. And yes it should be create-plugin tool